content/_data/authors/ewelinawilkosz.adoc

```diff
@@ -1,4 +1,11 @@
 ---
 name: "Ewelina Wilkosz"
 github: "ewelinawilkosz"
+twitter: "WilkoszEwelina"
 ---
+
+Jenkins Contributor since 2017, when she got involved in Jenkins Configuration as Code Plugin development.
+Voted Most Valuable Contributor in 2018.
+She has 14 years of experience in IT, working as a CI/CD consultant since the beginning of 2017.
+In that role she’s trying to solve numerous issues Jenkins issues are facing daily - as  users, administrators, maintainers. 
```

Comment:
Maybe something like
```suggestion
In that role she’s trying to solve numerous issues Jenkins community is facing daily - as  users, administrators, maintainers. 
```
(same in the article)

Reply:
or like below? I'll be ok with any of those two
```suggestion
In that role she’s trying to solve numerous issues Jenkins users are facing daily - as developers, administrators, maintainers.
```